aspnetcore/razor-pages/ui-class.md

```diff
@@ -4,14 +4,184 @@ author: Rick-Anderson
 description: Explains how to create reusable Razor UI using partial views in a class library in ASP.NET Core.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
-ms.date: 08/22/2019
+ms.date: 09/21/2019
 ms.custom: "mvc, seodec18"
 uid: razor-pages/ui-class
 ---
 # Create reusable UI using the Razor class library project in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
+::: moniker range=">= aspnetcore-3.0"
+
+Razor views, pages, controllers, page models, [Razor components](xref:blazor/class-libraries), [View components](xref:mvc/views/view-components), and data models can be built into a Razor class library (RCL). The RCL can be packaged and reused. Applications can include the RCL and override the views and pages it contains. When a view, partial view, or Razor Page is found in both the web app and the RCL, the Razor markup (*.cshtml* file) in the web app takes precedence.
+
+This feature requires [!INCLUDE[](~/includes/2.1-SDK.md)]
```

Comment:
3.0 ?

Reply:
## Use the Razor UI library from a Razor Pages project

Is missing from the 3.0 version.